tests/integration/docusaurus/connecting_to_your_data/cloud/pandas_s3_python_example.py

```diff
@@ -0,0 +1,100 @@
+from ruamel import yaml
+
+import great_expectations as ge
+from great_expectations.core.batch import BatchRequest, RuntimeBatchRequest
+
+context = ge.get_context()
+
+datasource_config = {
+    "name": "my_s3_datasource",
+    "class_name": "Datasource",
+    "execution_engine": {"class_name": "PandasExecutionEngine"},
+    "data_connectors": {
+        "default_runtime_data_connector_name": {
+            "class_name": "RuntimeDataConnector",
+            "batch_identifiers": ["default_identifier_name"],
+        },
+        "default_inferred_data_connector_name": {
+            "class_name": "InferredAssetS3DataConnector",
+            "bucket": "<YOUR_S3_BUCKET_HERE>",
+            "prefix": "<BUCKET_PATH_TO_DATA>",
+            "default_regex": {
+                "group_names": ["data_asset_name"],
+                "pattern": "(.*)\\.csv",
+            },
+        },
+    },
```

Comment:
Question on this. This datasource has both a runtime and inferred data connector. The scripts show only the runtime use case. I think other guides show both use cases. I'm not sure if this guide should show both or if it's simpler just with one. What do you think? Should we chat about it?

Reply:
Hi @Aylr I'd like to double-check my understanding on this comment.  I highlighted (what I think is) the `inferred_data_connector` example below. Is this what you were referring to?  

On the flip side, I think we could be more explicit in what is meant by the s3 path ("it starts with s3://<bucketname> etc etc") but maybe the guide would be more clean without it